src/components/link/link.stories.js

```diff
@@ -49,7 +50,7 @@ function makeStory(name, themeSelector) {
         tooltipMessage={ tooltipMessage }
         tooltipPosition={ tooltipPosition }
         tooltipAlign={ tooltipAlign }
-        onClick={ action('click') }
+        onClick={ hasOnClick && action('click') }
```

Comment:
When `hasOnClick` is `false`, then `hasOnClick && action('click')` will evaluate to `false` which will be passed to the `onClick` prop of `<Link>`.

In `<Link>`, the `propType` of `onClick` is `func`:

* https://github.com/Sage/carbon/blob/180f948/src/components/link/link.component.js#L141

So will this cause a `propType` validation failure if we pass `false` when it's expecting a function?

Reply:
Yeah good point have updated it to pass undefined if false

Reply:
Perfect, thanks :white_check_mark: